docs/latest/modules/en/pages/k8s-suse-rancher-prime.adoc

```diff
@@ -375,7 +375,7 @@ sts service-token create --name suse-observability-extension --roles stackstate-
 The deployment of the SUSE Observability agent requires the following system privileges:
 
 . `hostPID: true`: This privilege is required to associate process identifiers (PIDs) with their corresponding control groups (cgroups). This association is essential for accurately mapping processes to their respective containers.
-. `hostNetwork: true`: this privilege is required to scrape open metrics data from other pods on the node.
+. `hostNetwork: true`:  This configuration is not mandatory. By default, the node agent runs with `hostNetwork: true` to facilitate scraping of open metrics data from all configured pods on the node without requiring additional network policies. If disabled, appropriate network policies must be defined to ensure the agent can access the necessary endpoints.
```

Comment:
Couldn't we include some network policies that by default allow for this behavior without having to resort to `hostNetwork: true`?

@craffit what do you think?